src/box/iproto_features.h

```diff
@@ -107,7 +112,7 @@ struct iproto_features {
  * `box.iproto.protocol_version` needs to be updated correspondingly.
  */
 enum {
-	IPROTO_CURRENT_VERSION = 8,
+	IPROTO_CURRENT_VERSION = 9,
```

Comment:
Will this patch be backported to 3.1.1? If so, shouldn't we reorder `FETCH_SNAPSHOT_CURSOR` and `IS_SYNC` so that Tarantool 3.1.1 would have 8 version of IPROTO and 3.2 would have 9?

Reply:
I'm planning to define the `FETCH_SNAPSHOT_CURSOR` feature and bump the protocol version to 9 in 3.1. It's okay because this feature is supported only in EE anyway.